sdk/src/main/java/io/dapr/client/DaprClientGrpcAdapter.java

```diff
@@ -114,11 +108,24 @@
   /**
    * Operation not supported for GRPC
    *
-   * TODO: Implement this since this IS supported.
    * @throws UnsupportedOperationException every time is called.
    */
   public <T> Mono<Void> invokeService(String verb, String appId, String method, T request) {
-    return Mono.error(new UnsupportedOperationException("Operation not supported for GRPC"));
```

Comment:
I have an implementation for this. It just needs to call into the other one, no need to duplicate code.